modules/sublimeBidAdapter.js

```diff
@@ -71,6 +73,19 @@ export const spec = {
         gdprApplies: gdpr.gdprApplies
       };
     }
+
+    const pbjs = {
+      bt: config.getConfig('bidderTimeout'),
+      ts: Date.now()
+    }
+
+    window.sublime = (typeof window.sublime !== 'undefined') ? window.sublime : {};
```

Comment:
Can we use `window.sublime ? window.sublime : {}` to avoid other possible accidents ?